### DISCUSSION TITLE
About free books on your website

### DISCUSSION THREAD

**Discussion by Mehrol911** [community/community#27843] [Other Feature Feedback, Questions, & Ideas] [answered]
Good day Dibya Chakravorty,

I want to ask one question. I see that there are free python books on your website, is it legal to use them for free. I mean you have a permission to put all of this to your website. I want to clarify whether its legal or not, if it's legal, i will start using it.

I am looking forward hearing from you.

Best Wishes,
Mehrol Bazarov

**Comment by EliasAfara** [comment, marked as answer]
It is legal to use them, no need to worry about any legal stuff. 

The website that Dibya maintains contains a [free books](https://pythonbooks.org/free-books/) section, check it out.

**Reply by Mehrol911** [reply]
Thank you a lot EliasAfara.

**Reply by baselmansour961** [reply]
> It is legal to use them, no need to worry about any legal stuff. 
> 
> The website that Dibya maintains contains a [free books](https://pythonbooks.org/free-books/) section, check it out.

**Comment by jfullstackdev** [comment]
same as open source license, the content is still copyright in a sense, you need to at least mention that it was from them, exact words,

but if you did copy it and claimed that it's your own, you run the risk of using copyright content without permission

**Reply by Mehrol911** [reply]
It is just self-learning no want copying. Thank you a lot. I got it
